wp-content/themes/core/components/share/js/share.js

```diff
@@ -48,7 +48,7 @@ const bindEvents = () => {
  */
 
 const socialShare = () => {
-	if ( ! el.container ) {
+	if ( el.container.length === 0 ) {
```

Comment:
Note that since we are no longer setting `el.container` to the first item in the node array, `! el.container` evaluates as `true` even when no elements are present.